src/framework/analysisState.ml

```diff
@@ -7,6 +7,15 @@ let should_warn = ref false
 (** Whether signed overflow or underflow happened *)
 let svcomp_may_overflow = ref false
 
+(** Whether a Use-After-Free (UAF) happened *)
+let svcomp_may_use_after_free = ref false
+
+(** Whether an invalid pointer dereference happened *)
+let svcomp_may_invalid_deref = ref false
+
+(** Whether an invalid memtrack happened *)
+let svcomp_may_invalid_memtrack = ref false
```

Comment:
This still treats use-after-free as equivalent to invalid-free, but as I mentioned above, that's not the case. A UAF constitutes as an invalid-deref of memory which has been freed, while the freeing itself was completely valid.

So UAF is a subclass of invalid-deref. But being a subclass means we can't soundly analyze that unless we also find every other invalid-deref, not just UAF.
Also, competing in valid-memsafety requires verifying all three properties simultaneously. If we don't analyze the other two, we also cannot output sound results. So I'm doubtful that we'll be able to do anything meaningful in the actual SV-COMP scenario.

Reply:
The issue with the correct classification of UAF as `invalid-deref` is addressed and should be now fixed by https://github.com/goblint/analyzer/pull/1123/commits/d49db71fee1a9ffc125cf373eaa0bf2dab0baa58